src/components/AuthForm.js

```diff
@@ -181,11 +188,25 @@ function extendUiSchemaWithHistory(uiSchema, history, clearHistory) {
 }
 
 export default class AuthForm extends Component {
+  props: {
+    session: SessionState,
+    history: string[],
+    setup: Function,
+    navigateToExternalAuth: Function,
+    clearHistory: Function,
+  };
+
+  state: {
+    schema: Object,
+    uiSchema: Object,
+    formData: Object,
+  };
+
   defaultProps = {
     history: []
   };
 
-  constructor(props) {
+  constructor(props: Object) {
```

Comment:
This would actually want to be `props: Props` and the Prop type being extracted from below, though having these inline is much less verbose...